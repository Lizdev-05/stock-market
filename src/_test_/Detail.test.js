import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import CompanyDetails from '../components/details/Details';
import store from '../redux/configStore';

const DetailsProvider = () => (
  <Provider store={store}>
    <CompanyDetails />
  </Provider>
);

describe('renders properly the company details page', () => {
  it('renders correctly', () => {
    const component = renderer.create(<DetailsProvider />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
