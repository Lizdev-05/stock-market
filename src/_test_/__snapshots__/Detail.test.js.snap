// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly the company details page renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="justify-content-center align-content-center mt-5 row"
  >
    <span
      aria-busy={true}
      aria-live="polite"
    >
      <span
        className="react-loading-skeleton"
        style={
          Object {
            "--base-color": "#00A9D8",
            "--highlight-color": "#f8f9f9",
          }
        }
      >
        ‌
      </span>
      <br />
      <span
        className="react-loading-skeleton"
        style={
          Object {
            "--base-color": "#00A9D8",
            "--highlight-color": "#f8f9f9",
          }
        }
      >
        ‌
      </span>
      <br />
      <span
        className="react-loading-skeleton"
        style={
          Object {
            "--base-color": "#00A9D8",
            "--highlight-color": "#f8f9f9",
          }
        }
      >
        ‌
      </span>
      <br />
    </span>
  </div>
</div>
`;
